tensorrt_llm/_torch/attention_backend/sparse/minimax_m3/backend.py

```
@@ -0,0 +1,1286 @@
+tensor ops with static shapes derived from those CPU-side scalars;
+no ``.item()`` or other GPU-CPU sync runs inside the forward
+functions. The algorithm captures cleanly under a CUDA graph and
+replays bit-identical output.
+"""
+
+from __future__ import annotations
+
+import functools
+import os
+from dataclasses import dataclass
+from typing import List, Optional, Tuple
```

Flake8 (7.3.0)

[error] 'typing.List' imported but unused

(F401)